[3.0.0 regression] error TS7016: Could not find a declaration file for module './adjacencyGraphs.json'

From an empty directory:

```console
$ npm i @zxcvbn-ts/language-common typescript

added 2 packages in 359ms

$ echo 'import * as zxcvbnCommonPackage from "@zxcvbn-ts/language-common";' > test.ts

$ npx tsc --strict test.ts 
node_modules/@zxcvbn-ts/language-common/dist/index.d.ts:1:29 - error TS7016: Could not find a declaration file for module './adjacencyGraphs.json'. '/tmp/test/node_modules/@zxcvbn-ts/language-common/dist/adjacencyGraphs.json.js' implicitly has an 'any' type.

1 import adjacencyGraphs from './adjacencyGraphs.json';
                              ~~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in node_modules/@zxcvbn-ts/language-common/dist/index.d.ts:1
```

Fails in `@zxcvbn-ts/language-common` 3.0.0 and 3.0.1; worked in 2.0.1.

--- a/packages/languages/common/src/index.ts
+++ b/packages/languages/common/src/index.ts
@@ -1,7 +1,8 @@
-import adjacencyGraphs from './adjacencyGraphs.json'
+import * as tempAdjacencyGraphs from './adjacencyGraphs.json'
 import passwords from './passwords.json'
 import diceware from './diceware.json'
 
 const dictionary = { passwords, diceware }
-
+// reinit the variable because the typescript compile will otherwise just reexport the json file which will break as it's a json file
+const adjacencyGraphs = tempAdjacencyGraphs
 export { dictionary, adjacencyGraphs }

